softlayer/vm/softlayer_hardware.go

```diff
@@ -639,3 +657,45 @@ func (vm *softLayerHardware) searchMounts() ([]Mount, error) {
 
 	return mounts, nil
 }
+
+func (vm *softLayerHardware) provisionBaremetal(server_id string, stemcell string, netboot_image string) (int, error) {
+	provisioningBaremetalInfo := bmscl.ProvisioningBaremetalInfo{
+		VmNamePrefix:     server_id,
+		Bm_stemcell:      stemcell,
+		Bm_netboot_image: netboot_image,
+	}
+	createBaremetalResponse, err := vm.baremetalClient.ProvisioningBaremetal(provisioningBaremetalInfo)
+	if err != nil || createBaremetalResponse.Status != 200 || createBaremetalResponse.Data.TaskId == 0 {
+		return 0, bosherr.WrapErrorf(err, "Failed to provisioning baremetal")
+	}
+
+	task_id := createBaremetalResponse.Data.TaskId
+	bslcommon.TIMEOUT = 120 * time.Minute
```

Comment:
This seems like a long timeout. Should we configure this into some config and perhaps this `120 * time.Minute` is the default.